src/mixins/java/org/spongepowered/common/mixin/core/server/level/ServerLevelMixin.java

```diff
@@ -441,6 +441,15 @@ public void save(@Nullable final ProgressListener progress, final boolean flush,
         this.impl$recentTickTimes[this.shadow$getServer().getTickCount() % 100] = postTickTime - this.impl$preTickTime;
     }
 
+    @Inject(method = "tick", at = @At("RETURN"))
```

Comment:
This is also out of scope for implementing the API portion of this, just so you know.